SECURITY.md

```diff
@@ -1,4 +1,4 @@
-<!-- BEGIN MICROSOFT SECURITY.MD V0.0.5 BLOCK -->
+[<!-- BEGIN MICROSOFT SECURITY.MD V0.0.5 BLOCK -->
```

Comment:
Er, was this supposed to change? Seems like it pasted in a random markdown link.

Reply:
Oh sorry, forgot to actually change the file😅